pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -283,6 +285,11 @@ clone_user_rev() {
     local url="$2"
     local rev="${3:-HEAD}"
 
+    if [ -n "$fetchLFS" ]; then
+        HOME=$TMPDIR
+        git lfs install
```

Comment:
What does this do?

Reply:
It will essentially add this:
```
# ~/.config/git/config
[filter "lfs"]
	clean = "git-lfs clean -- %f"
	process = "git-lfs filter-process"
	required = true
	smudge = "git-lfs smudge -- %f"
```
Which changes the behavior of git to substitute the git lfs pointer files to lfs file.